src/Draco.Editor.Web/app/logo-handler.js

```diff
@@ -0,0 +1,48 @@
+import {get} from 'https';
```

Comment:
`get()` is not very readable, here you could have import the whole module and do `http.get`